File: qtile/modules/apps.py
"""
Aliases for common applications
"""

apps = {
  "terminal":                   "alacritty",
  "web_browser":                "firefox",
  "file_manager":               "alacritty --class FileManager -e ranger",
  "local_music_player":         "alacritty --class Ncmpcpp -e ncmpcpp",
  "streaming_music_player":     "spotify",
  "program_launcher":           "rofi -theme ~/.config/rofi/app_menu.rasi -show drun",
  "shutdown_menu":              "",
  "text_editor":                "alacritty -e nvim",
  "code_editor":                "codium"
}

File: awesome/config/apps.lua
local _M = {
    terminal        = "alacritty",
    editor          = "nvim",
    browser         = "firefox",
    text_editor     = "codium",
    music_streamer  = "spotify",
    file_manager    = "thunar",
}

_M.editor_cmd = _M.terminal .. " -e " .. _M.editor
_M.manual_cmd = _M.terminal .. " -e man awesome"

return _M
